client/src/components/restaurants/BlissBee.jsx

```diff
@@ -0,0 +1,63 @@
+// React imports
+import React from "react";
+import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+// Style and Component imports
+import "../../App.css";
+import Sidebar from "../Sidebar";
+import Header from "../Header";
+import RestaurantInfo from "../RestaurantInfo";
+import BlissBeeMap from "../restaurantMaps/BlissBeeMap";
+
+// useState and restaurant info declarations
+let restInfo;
+function GreatHarvest(props) {
+  const [restName, setRestName] = useState("");
+  const [restHours, setRestHours] = useState("");
+  const [restAddress, setRestAddress] = useState("");
+  const [restNoteOne, setRestNoteOne] = useState("");
+  const [restNoteTwo, setRestNoteTwo] = useState("");
+  const [restPhone, setRestPhone] = useState("");
+
+  // On page load fetches specific restaurant info fom server and stores json in restInfo
+  // states are updated with restaurant info
+  useEffect(() => {
+    fetch("/api/bliss-bee")
```

Comment:
Since this is the only line that changes in your other Restaurant components you could make it so the fetch can be changed programmatically to update your other fields. That way you could just have one restaurant component.